### PR TITLE
ListItemTag with MD3 Theme

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes 4372 .

#### Changes proposed in this Pull Request:
- Update ListItemTag
- Copy doc file in MD3 Theme
- 

#### Screenshots / Screen Recording (if applicable)
Screenshot 2023-09-01 at 12 25 32 PM

Screenshot 2023-09-01 at 12 25 59 PM

#### To Test:

- cd MD3
- run yarn start ios

#### Any specific feedback you are looking for?
-

### CODE REVIEW

**Comment by huayunh** [comment]
why did the list item tags span full width in your screenshot?

**Comment by ektaghag-eaton** [comment]
> why did the list item tags span full width in your screenshot?

Not sure I think it is because I am wrapping it inside the Text component, let me debug more and fix it.

**Comment by ektaghag-eaton** [comment]
> why did the list item tags span full width in your screenshot?
Fixed. Please check.
Screenshot 2023-09-05 at 5 39 20 PM

File: components/src/md3/ListItemTag/ListItemTag.tsx [hunk 1]
```diff
@@ -78,8 +78,9 @@ const listItemTagStyles = (
             fontWeight: '700',
             overflow: 'hidden',
             lineHeight: 16,
-            fontSize: 10,
-            letterSpacing: 1,
+            fontSize: 12,
+            letterSpacing: 2,
+            textTransform: 'uppercase',
```
[the remark this answers is not in the document]
Comment (ektaghag-eaton) [reply components/src/md3/ListItemTag/ListItemTag.tsx]: Fixed. Please check.

**Comment by huayunh** [comment]
visually seems fine (didn't check the code)

**Comment by ektaghag-eaton** [comment]
@JeffGreiner-eaton  or @daileytj  can anyone please review these changes on Android?

**Comment by daileytj** [comment]
looks good on android
Screenshot_1694004159

**Comment by ektaghag-eaton** [comment]
> looks good on android Screenshot_1694004159

Thank you!!

**Pull request merged**